server/channels/jobs/batch_worker.go

```diff
@@ -73,10 +84,14 @@ func (worker *BatchWorker) Run() {
 
 // Stop interrupts the worker even if the migration has not yet completed.
 func (worker *BatchWorker) Stop() {
+	worker.stateMut.Lock()
+	defer worker.stateMut.Unlock()
+
 	// Set to close, and if already closed before, then return.
```

Comment:
Is that comment still valid?

Reply:
Yep, it is. The variable just changed from an atomic bool, to a normal bool.